Dockerfile.cross

```
@@ -14,9 +14,10 @@ ENV PKG_CACHE_PATH=/fetched
 
 RUN git clone -b master --single-branch https://github.com/vercel/pkg-fetch.git /build
 WORKDIR /build
+RUN yarn add babel-cli
 RUN yarn && yarn run babel
 RUN mkdir -p /fetched
 
 RUN MAKE_JOB_COUNT=${NPROC} yarn run bin ${PKG_NODE} ${PKG_OS} ${PKG_ARCH}
```

```suggestion
RUN MAKE_JOB_COUNT=${NPROC} npm start --force-build --node-range ${PKG_NODE} --arch ${PKG_ARCH}
```

With latest version 3.0.1 the execute command is also changed